enhancements/hypershift/multi-arch-hypershift.md

```diff
@@ -0,0 +1,195 @@
+---
+title: hypershift-for-multiple-architectures
+authors:
+  - "@jaypoulz"
+  - "@mkumatag"
+  - "@deepsm007"
+  - "@jeffdyoung"
+  - "@bryan-cox"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+api-approvers:
+  - None
+creation-date: 2022-07-08
+last-updated: 2022-07-08
+tracking-link:
+  - [HyperShift Epic](https://issues.redhat.com/browse/MULTIARCH-2060)
+see-also:
+  - [Heterogeneous Enhancement](https://github.com/openshift/enhancements/pull/1014)
+---
+# HyperShift Across Multiple Architectures
+
+Expanding upon our work towards OpenShift working across nodes of different
+architectures, HyperShift opens up new doors for us. With HyperShift, we can
+maximize cost-efficiency by running our orchestration workloads on competitively
+priced instance types, regardless of architecture. Other objectives includ
+enabling operators whose availability is tied to an architecture to still be
+able to run regardless of the control plane architecture.
+
+## Summary
+
+Enabling HyperShift across Multiple Architectures starts by extending the
+platforms and architectures it can currently target as compute nodes. This
+enables clusters to benefit from HyperShift's scaling on x86 while still
+being able to target workloads on IBM Power, IBM zSystems, and ARM.
+
+Another driver major driver is the cost-benefit of being able to schedule
```

Comment:
I think there's a typo here, probably should just start with _Another major driver_?